### DISCUSSION TITLE
Tauon last release on Windows 11.

### DISCUSSION THREAD

**Discussion by Zutyosh** [Taiko2k/Tauon#1359] [Q&A] [answered]
Hey, I love Tauon, but I had to switch recently to Windows and I realize that there are no more setup releases for the latest versions and I don't know how to compile an installation myself with the source code.

Can anyone help me?

**Comment by C0rn3j** [comment, marked as answer]
You can either build from source - https://github.com/Taiko2k/Tauon/wiki/Building-for-Windows

Or grab a prebuilt dev release from the latest commit, just tap on the checkmark, open Windows action, Summary, and the download is there.

Windows CI builds literally just got merged 2 hours ago, so you're in luck :)

The prebuilt images have some issues tracked here, if you run into anything else, do report it - https://github.com/Taiko2k/Tauon/issues/1315

If you happen to have Spotify Premium, testing the integration would be welcome.

**Reply by Zutyosh** [reply]
Okay, I see thanks to you !
